rosidl_typesupport_connext_c/resource/msg__type_support_c.cpp.em

```diff
@@ -60,41 +61,50 @@ extern "C"
 // Forward declare the get type support function for this type.
 ROSIDL_GENERATOR_C_EXPORT_@(pkg)
 const rosidl_message_type_support_t *
-ROSIDL_GET_TYPE_SUPPORT_FUNCTION(@(pkg), @(subfolder), @(msg))(void);
+  ROSIDL_GET_TYPE_SUPPORT_FUNCTION(@(pkg), @(subfolder), @(msg))(void);
```

Comment:
This is also not right, since it's a function declaration. Should we wrap it in an uncrustify nolint or just live with the incorrect indentation?